packages/database/index.node.ts

```diff
@@ -25,6 +25,28 @@ import * as TEST_ACCESS from './src/api/test_access';
 import { isNodeSdk } from '@firebase/util';
 import './src/nodePatches';
 
+/**
+ * A one off register function which returns a database based on the app and
+ * passed database URL.
+ * 
+ * @param app A valid FirebaseApp-like object
+ * @param url A valid Firebase databaseURL 
+ */
+export function initStandalone(app, url) {
+  return {
+    instance: RepoManager.getInstance().databaseFromApp(app, url),
+    namespaces: {
```

Comment:
Should 'namespaces' be 'namespace' ?

Reply:
'namespace' makes more sense to me.  These are the properties that appear in the firebase.database namespace, right?  And the actual properties are mostly classes (I wouldn't call any of them a 'namespace').  So 'namespace' makes more sense.  But since it's internal goo I don't care strongly.

Reply:
Okay! That sounds fine to me.